src/views/projects/CloneButton.svelte

```diff
@@ -92,7 +92,7 @@
   <svelte:fragment slot="modal">
     <div class="dropdown">
       <div class="clone-url-wrapper">
-        <div class="clone-url" name="rad-clone-url">{radCloneUrl}</div>
+        <div class="clone-url" id="rad-clone-url">{radCloneUrl}</div>
```

Comment:
Did svelte-check complain about this? Are we even using this for anything?

Reply:
Yeah `svelte-check` started complaining
And we use it for the `label` tag a bit further down